src/Containers/Microsoft.NET.Build.Containers/Registry/Registry.cs

```diff
@@ -122,6 +122,11 @@ private static string DeriveRegistryName(Uri baseUri)
     /// </summary>
     public bool IsGithubPackageRegistry => RegistryName.StartsWith("ghcr.io", StringComparison.Ordinal);
 
+    /// <summary>
+    /// Is this registry the public Microsoft Container Registry.
+    /// </summary>
+    public bool IsMCR => RegistryName.Equals("mcr.microsoft.com", StringComparison.Ordinal);
```

Comment:
nit: By the naming convention, it should be `IsMcr` (I know we didn't follow it everywhere, but we can follow it for the new code :))